documentation/Dumping.md

```diff
@@ -0,0 +1,16 @@
+# Dumping Games
+
+Dumping your games is the only legal way of obtaining commercial games, and the only allowed way to get games for Vita3K.
+
+However, if you're after homebrew games, you can search online or visit [VitaDB](https://vitadb.rinnegatamante.it/#/). Be warned that some homebrew apps may contain potentially unwanted software like malware.
+
+# Dumping methods
+
+There are four ways to dump Vita games: MaiDump, [NoNpDrm](https://github.com/TheOfficialFloW/NoNpDrm), Vitamin and FAGdec. However, MaiDump and Vitamin are not supported by Vita3K.
+
+The games should be in a .zip or .vpk format if you want to install them onto the emulator, or if you prefer to copy them yourself, you can drag and drop the game folder in your `pref_path/ux0/app` folder. *You do not need to do this for NoNpDrm/.pkg files.**
```

Comment:
again zip or pkg for comercial, vpk only for homebrews.

Reply:
fixed